src/api/controllers/profiles.js

```diff
@@ -31,18 +31,28 @@ export default class ProfilesController {
   static async updateProfile(req, res) {
     const { body, user } = req;
     try {
+      if (Object.keys(body) < 1) return res.status(400).send({ message: 'Nothing Changed to your Profile' });
       const updatedUser = await User.update(
         { ...body },
         { where: { id: user.id } },
       );
-
-      if (!updatedUser[0]) {
-        return res.status(404).json({ message: `Could not find user with id: ${user.id}` });
-      }
-
       return res.status(200).json({ user: { updatedUser } });
     } catch (error) {
       return res.status(500).json({ error: `${error}` });
     }
   }
+
+  /**
+   * @Author - Audace Uhiriwe
+   * @param  {object} req
+   * @param  {object} res
+   * @return {object} returns an object containing the user's profiles
+   */
+  static async getAllProfile(req, res) {
+    const fetchedProfile = await User.findAll({ attributes: ['username', 'firstName', 'lastName', 'bio', 'image'] });
+    if (!fetchedProfile[0]) return res.status(404).send({ error: 'No Users Profiles found!' });
```

Comment:
This is not a client error, it should not be 404.

Reply:
@eliemugenzi Thanks! i'm going to fix it